pkg/management/postgres/instance.go

```diff
@@ -199,6 +200,12 @@ type Instance struct {
 
 	// tablespaceSynchronizerChan is used to send tablespace configuration to the tablespace synchronizer
 	tablespaceSynchronizerChan chan map[string]apiv1.TablespaceConfiguration
+
+	// StatusTLS enables TLS on the status port used to communicate with the operator
+	StatusTLS bool
```

Comment:
IMO since "status" is so overloaded already, this would be better clarified.
How about `StatusPortTLS`?